models/Tree.php

```diff
@@ -294,7 +297,7 @@ public function removeNode($softDelete = true, $currNode = true)
                     }
                 }
             }
-            if (!empty($this->nodeRemovalErrors)) {
+            if ($this->nodeRemovalErrors) {
                 if ($currNode) {
                     $this->active = false;
```

Comment:
`$this->active = false` should not be done only in case of errors and therefore it might be unnecessary at this point, see below